client/.gitignore

```diff
@@ -28,3 +28,6 @@ package-lock.json
 /src/samples
 .env
 /src/lib/pushClientEvent.ts
+/public/
+!/public/uv/
```

Comment:
We're ok with not pushing up the UV config? This would impact the user's ability to get the IIIF viewer working.

Reply:
This says to ignore the public folder, but DON'T ignore the UV config. (The UV config will still be in the repo)

Reply:
Welp, you learn something knew everyday. Merging!